nixos/tests/galene.nix

```diff
@@ -208,4 +208,136 @@ in
       '';
     }
   );
+
+  stream = makeTest (
```

Comment:
Maybe we should use `runTest`? (see https://github.com/NixOS/nixpkgs/issues/386873).

Reply:
Ohh, I didn't realise that applied to test matrices too… Will do.